WeatherAppForBadWeather/ViewController.m

```diff
@@ -14,6 +14,10 @@ - (void)viewDidLoad {
     [super viewDidLoad];
 
     // Do any additional setup after loading the view.
+    
+    NSLog(@"The weather is still bad 😨");
+    NSLog(@"But maybe it will get better? 🤗");
```

Comment:
I don't think that's a good idea.

I thought we agreed to use printf in this project.